config/version-number-rules.xml

```diff
@@ -11,6 +11,12 @@
     </ignoreVersion>
   </ignoreVersions>
   <rules>
+    <rule groupId="com.github.caciocavallosilano" artifactId="cacio-tta">
+      <ignoreVersions>
+        <!-- Version 1.17 is built with Java17 bytecode, we need Java11  -->
+        <ignoreVersion type="exact">1.17</ignoreVersion>
```

Comment:
Shouldn't this be version 1.17 or later? I doubt they will downgrade to Java 11 afterwards.

Reply:
I tried using version 1.17. It works on my PC but fails in CI because version 1.17 is not compatible with Java11.

Reply:
@pbludov My comment is on the following section:
> type="exact"

Isn't this specifying the version of `caciocavallosilano`? It is saying we will only ignore that version of `caciocavallosilano`, right?

Reply:
Yes. I'd like to see what the next version of `caciocavallosilano` will be.
After that, it will be clear how to compose the pattern.

Reply:
Based on their history, won't it most likely be 18+ if not 2? https://mvnrepository.com/artifact/com.github.caciocavallosilano/cacio-tta
Why can't we just assume that now?

Reply:
The next release should be 1.21. But it could be 2.0. 
I don't know how to write a readable template that will pass 1.11.1, but reject 1.21.
Please suggest.

Reply:
Unless they plan to downgrade Java version, we should just drop the version completely and ignore all versions. If the current one we are on is the last Java 11 supports, then we will have to stick with it always. So it should be similar to what we have for `maven-release-plugin` in this same file.

Reply:
done.